conf/defaults.ini

```diff
@@ -333,6 +333,9 @@ auto_assign_org_id = 1
 # Default role new users will be automatically assigned (if auto_assign_org above is set to true)
 auto_assign_org_role = Viewer
 
+# Skip forced assignment of OrgID 1 or 'auto_assign_org_id'  for social logins
+skip_org_role_update_sync = false
+
```

Comment:
First, sample.ini needs the same update, but out-commented. 

Second, since this is a setting only applied for OAuth feels like the setting should be under the section `[auth]`, rather than under `[users]`, and be named `oauth_skip_org_role_update_sync`. With these changes need to update the documentation/configuration.md as well.

Thoughts?

Reply:
Sounds reasonable, I'll apply this tomorrow if no one else has a different opinion?
Not sure if `oauth` or `social` fits better as a prefix but to be honest, I've neither a strong opinion nor a 'good feeling' about code style and structure in Grafana hence I'd be glad to follow your lead.

Reply:
I renamed and relocated the setting as requested. For now I pushed it as a second commit but I can squash them as soon as the final location is found

Reply:
@achatterjee-grafana I also updated the docs and moved the corresponding part to the `auth` section of course. Apart from that I didn't change anything besides the header obviously